utils/download-toolchain.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
Issue: We've made the decision as a team to use python for our codebase, and this is introducing a new bash script, adding to technical debt. We should stick to the decision to use python unless we discuss as a team to change the decision.